docs/src/design/chiplets/main.md

```diff
@@ -0,0 +1,171 @@
+# Chiplets
+
+The Chiplets module reduces the number of columns required by the execution trace by stacking the execution traces of 3 chiplets that are expected to generate significantly fewer rows than the other VM processors (the decoder, stack, and range checker).
+
+## Chiplet components
+
+The chiplets are:
+
+- [Hash Processor](./hasher.md) (17 columns; degree 8)
+- [Bitwise Processor](./bitwise.md) (14 columns; degree 6)
+- [Memory Processor](./memory.md) (14 columns; degree 6)
```

Comment:
Sure. In general, I replaced every reference to "co-processor" or "coprocessor", but I didn't update the references to "Processor" because it's a much more time-consuming replacement. I'll do a pass to update manually, but there may be some missed.